### PR TITLE
Update for use with new cryptobox

### PR DESCRIPTION
**Pull request by daehn**
### What's in this PR?

* Bump `wire-ios-sync-engine` and `wire-iso-data-model` to the new versions using the new cryptobox and adapt the changed API to create a `ZMUserSession`.

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -14,13 +14,13 @@ github "wireapp/ocmock" "v3.1.3"
...
 github "wireapp/wire-ios-ziphy" "3.0.2"
 github "wireapp/wire-ios-link-preview" "3.7.0"
 github "wireapp/wire-ios-utilities" "10.30.3"
 github "wireapp/wire-ios-images" "4.18.5"
 github "wireapp/wire-ios-transport" "9.0.1"
 github "wireapp/wire-ios-protos" "6.3.0"
-github "wireapp/wire-ios-data-model" "12.1.3"
-github "wireapp/wire-ios-sync-engine" "51.6.0"
+github "wireapp/wire-ios-data-model" "13.0.0"
+github "wireapp/wire-ios-sync-engine" "d95cc45657b590d1c9f0e2249576c4e3856e02e3"
```
Comment (mikeger) [inline Cartfile.resolved]: I think we need to wait for release then?

File: Cartfile.resolved [hunk 2]
```diff
@@ -14,13 +14,13 @@ github "wireapp/ocmock" "v3.1.3"
...
 github "wireapp/wire-ios-ziphy" "3.0.2"
 github "wireapp/wire-ios-link-preview" "3.7.0"
 github "wireapp/wire-ios-utilities" "10.30.3"
 github "wireapp/wire-ios-images" "4.18.5"
 github "wireapp/wire-ios-transport" "9.0.1"
 github "wireapp/wire-ios-protos" "6.4.0"
-github "wireapp/wire-ios-data-model" "12.1.3"
-github "wireapp/wire-ios-sync-engine" "51.6.1"
+github "wireapp/wire-ios-data-model" "13.0.0"
+github "wireapp/wire-ios-sync-engine" "d95cc45657b590d1c9f0e2249576c4e3856e02e3"
```
Comment (marcoconti83) [inline Cartfile.resolved]: This should be changed to the new zmessaging version

**Comment by daehn** [comment]
@marcoconti83 @mikeger please re-review

**Comment by daehn** [comment]
Travis fails because it can't find some CocoaPods, looks similar to why it fails on bob

> 'AFNetworking/UIImageView+AFNetworking.h' file not found

**Comment by mikeger** [comment]
@daehn no worries, this issue is also on develop. I am on it.

**Pull request merged**